src/tools/discovery_parser/main.cpp

```diff
@@ -77,11 +77,11 @@ struct DiscoveryMap : std::enable_shared_from_this< DiscoveryMap >
   {
     auto ogX = x, ogY = y;
     int col = ( mapIndex % ( int ) ( ( float ) img.width / ( float ) tileWidth ) );
-    int row = ( mapIndex / ( ( float ) img.width / ( float ) tileWidth ) );
+    int row = static_cast<int>(( mapIndex / ( ( float ) img.width / ( float ) tileWidth ) ));
     x = ( x / 2048.f ) * ( float ) tileWidth;
     y = ( y / 2048.f ) * ( float ) tileWidth;
-    int tileX = ( col * ( float ) tileWidth ) + x;
-    int tileY = ( row * ( float ) tileWidth ) + y;
+    int tileX = static_cast<int>((col * (float)tileWidth) + x);
+    int tileY = static_cast<int>(( row * ( float ) tileWidth ) + y);
```

Comment:
mixing C-style casts with static_cast
see: `( float ) tileWidth`
styling, spaces between ( ) and < >